Dockerfile

```diff
@@ -50,9 +50,16 @@ RUN mkdir -p /usr/src/criu \
 
 # setup a playground for us to spawn containers in
 ENV ROOTFS /busybox
-RUN mkdir -p ${ROOTFS} \
-    && curl -o- -sSL 'https://github.com/docker-library/busybox/raw/a0558a9006ce0dd6f6ec5d56cfd3f32ebeeb815f/glibc/busybox.tar.xz' | tar xfJC - ${ROOTFS}
+RUN mkdir -p ${ROOTFS}
 
+RUN case $(dpkg --print-architecture) in \
+    arm64) \
+      curl -o- -sSL 'https://github.com/docker-library/busybox/raw/23fbd9c43e0f4bec7605091bfba23db278c367ac/glibc/busybox.tar.xz' | tar xfJC - ${ROOTFS} \
+      ;; \
+    *) \
+      curl -o- -sSL 'https://github.com/docker-library/busybox/raw/a0558a9006ce0dd6f6ec5d56cfd3f32ebeeb815f/glibc/busybox.tar.xz' | tar xfJC - ${ROOTFS} \
+      ;; \
+    esac
```

Comment:
This should be a script that both this and `helpers.bash` call into. No point in replicating this logic in more than one place.

Reply:
@cyphar 
Hi, I have moved all the multi-arch related code into 'tests/integration/multi-arch.bash'.
Please check it.
Thanks.